webapp/package.json

```diff
@@ -6,22 +6,24 @@
   "scripts": {
     "build": "webpack --mode=production",
     "lint": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet",
-    "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix"
+    "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix",
+    "test": "jest --forceExit --detectOpenHandles --verbose"
   },
   "author": "",
   "license": "",
   "devDependencies": {
-    "babel-core": "^6.26.3",
+    "@babel/core": "^7.5.0",
```

Comment:
Can you remove the unrelated dependency updates? We should handle these in a separate PR pulling in my changes from: https://github.com/mattermost/mattermost-plugin-demo/pull/42

Reply:
Aha, that's where they came from! Thanks, @crspeller -- @kaakaa, I can get these merged to `mattermost-plugin-starter-template`, and then a merge from upstream should resolve this concern.

Reply:
https://github.com/mattermost/mattermost-plugin-starter-template/pull/53 is up for review.

Reply:
Thanks @crspeller and @lieut-data.

I've merged it from upstream 👍